src/sentry/models/release.py

```diff
@@ -368,6 +368,27 @@ def is_valid_version(value):
             or value.lower() == "latest"
         )
 
+    @staticmethod
+    def is_semver_version(release):
```

Comment:
If you want to know whether a release follows semver, you can just check `release.package is not None`. If any semver field is filled, the release is semver. I don't think we need to rely on reparsing the version here.

I think this could also just be a property on the release rather than a `@staticmethod`. 

However, if you want this method to tell whether a project is using semver, it might be more useful to do `Release.objects.filter_to_semver().filter(organization=<org_id>, projects__in=[myproject]).exists()` (might be useful to filter to releases created in the last `<period of time>`)

Reply:
I agree with you we should probably have `is_semver` property so that we know if a release object follows semver but this function serves a different purpose; Its a static method that takes a release string (not a release object) and returns whether that release string follows semver

Reply:
Added the property + the changes we discussed on Friday in regards to checking for the latest ten releases if the follow semver or not

Reply:
Right, I think I left this comment before we discussed via hangouts. It might make sense to rename `release` to `version` to make it clearer